src/test/java/io/strimzi/kafka/metrics/KafkaMetricsCollectorTest.java

```diff
@@ -20,19 +18,19 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
-import java.util.HashMap;
 import java.util.LinkedHashMap;
 import java.util.Map;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertInstanceOf;
 
 public class KafkaMetricsCollectorTest {
-
     private final MetricConfig metricConfig = new MetricConfig();
     private final Time time = Time.SYSTEM;
     private Map<String, String> tagsMap;
     private Labels labels;
+    private static final String METRIC_PREFIX = "kafka.server";
```

Comment:
We typically put the `static final` fields above all other fields.

Reply:
Noted and moved